sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/utils/JdbcUtils.java

```diff
@@ -0,0 +1,119 @@
+package org.apache.beam.sdk.schemas.utils;
+
+import org.apache.beam.sdk.schemas.LogicalTypes;
+import org.apache.beam.sdk.schemas.Schema;
+import org.joda.time.Instant;
+
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Types;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JdbcUtils {
+
+    private static final Map<Integer, String > RS_TYPE_MAPPING = new HashMap<>();
```

Comment:
https://github.com/shehzaadn-vd/vend-beam/blob/abda92c62f51d3e8c123c3d54ccb2fa24d29f924/sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/utils/JdbcUtils.java#L16-L43

I don't think these lines are necessary if you use the [JDBCType](https://docs.oracle.com/javase/8/docs/api/index.html?java/sql/JDBCType.html) enum instead.

Reply:
Done